Unreal/SystemLocalized/det/Ceremony.det

```diff
@@ -12,11 +12,11 @@ LevelEnterText="Betrete den Zeremonienflügel"
 
 [TranslatorEvent15]
 ; EN: Message="The inner ceremonial chambers. Here the water god's serpents reside to accept the flesh of the weak to be devoured. Only a warrior who swims like the serpent shall escape their clutches."
-Message="Die inneren Zeremonienkammern. Hier leben die Schlangen des Wassergottes, die darauf heischen, das Fleisch der Schwachen zu verschlingen. Nur ein Krieger, der selbst wie die Schlange schwimmt, kann sich ihren Klauen entrinnen."
+Message="Die inneren Zeremonienkammern. Hier leben die Schlangen des Wassergottes, die darauf heischen, das Fleisch der Schwachen zu verschlingen. Nur ein Krieger, der selbst wie die Schlange schwimmt, kann ihren Krallen entrinnen."
 
 [TranslatorEvent0]
 ; EN: Message="Sgt. William Bradford: My entire squad has been killed by these salamander things. There's no ammo left and I've been holding up at this shrine with one of the native priests for days. While he prays I try and keep a watchful eye at the door."
-Message="Sgt. William Bradford: Mein ganzer Trupp wurde von diesen Salamanderviechern getötet. Ich habe keine Munition mehr und verstecke mich seit Tagen bei einem einheimischen Priester in diesem Schrein. Er betet, und ich behalte die Tür im Auge."
+Message="Sgt. William Bradford: Mein ganzer Trupp wurde von diesen Salamanderbiestern getötet. Ich habe keine Munition mehr und verstecke mich seit Tagen bei einem einheimischen Priester in diesem Schrein. Er betet, und ich behalte die Tür im Auge."
```

Comment:
Message="Unteroffizier William Bradford: Mein ganzer Trupp wurde von diesen Salamanderbiestern getötet. Ich habe keine Munition mehr und verstecke mich seit Tagen bei einem einheimischen Priester in diesem Schrein. Er betet, und ich behalte die Tür im Auge."